XMLDevourer.Tests/UtilsTests.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using Xunit;
+using XMLDevourer.Utils;
+
+namespace XMLDevourer.Tests
+{
+    public class UtilsTests
+    {
+        /*
+        Incomplete tests on IsPrime
+        */
+        [Fact]
+        public void IsPrimeTest()
```

Comment:
This never checks odd numbers that are **not** prime. Seems like a resonable test to do :thinking: